src/Sylius/Component/Core/Taxation/Applicator/OrderItemUnitsTaxesApplicator.php

```diff
@@ -57,13 +57,7 @@ public function __construct(CalculatorInterface $calculator, AdjustmentFactoryIn
     public function apply(OrderInterface $order, ZoneInterface $zone)
     {
         foreach ($order->getItems() as $item) {
-            $quantity = $item->getQuantity();
-            if (0 === $quantity) {
-                continue;
-            }
```

Comment:
I completely agree with removing this pointless check. I had tried to keep it consistent with the other tax applicator `Sylius\Component\Core\Taxation\Applicator\OrderItemsTaxesApplicator`. I suggest you update that one too. :)